test/scripts/e2e-cp.sh

```diff
@@ -115,7 +115,6 @@ if [ "$OVN_GATEWAY_MODE" == "local" ]; then
   if [ "$KIND_IPV6_SUPPORT" == true ]; then
     SKIPPED_TESTS+="Should be allowed to node local host-networked endpoints by nodeport services|\
 EgressQoS validation|\
-e2e br-int flow monitoring export validation|\
```

Comment:
wait this test was failing because of EIP? could you explain how?

Reply:
ah nvm the entire process wasn't coming up for this test which probably restarts the ovnk-nodes or something?
```
ovnkube.go:136] failed to run ovnkube: failed to start node network controller: failed to start default node network controller: failed to run egress IP controller: failed to create IP rule for node IPs: file exists 
```
?